GIT-VERSION-GEN

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 GVF=GIT-VERSION-FILE
-DEF_VER=v2.30.0-rc0
+DEF_VER=v2.30.0-rc0.vfs.0.0
```

Comment:
@mjcheetham I never include the `-rc0` because I think that the version parser will stumble over it.

Reply:
Updated and re-rebased to remove the `-rc0` part.